test/org/traccar/protocol/OpenGtsProtocolDecoderTest.java

```diff
@@ -0,0 +1,21 @@
+package org.traccar.protocol;
+
+import org.junit.Test;
+import org.traccar.ProtocolTest;
+
+public class OpenGtsProtocolDecoderTest extends ProtocolTest {
+
+    @Test
+    public void testDecode() throws Exception {
+
+        OpenGtsProtocolDecoder decoder = new OpenGtsProtocolDecoder(new OpenGtsProtocol());
+
+        verifyNull(decoder, request(
+                "/?bat=0"));
```

Comment:
Is this a real test case?

Reply:
Well, I don't see situation when id actually not coming, so we do not need to cover such possibility? Cause I don't have protocol documentation, I couldn't say for sure.